apps/factory/src/components/prompt_view_arrow.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { Stack, Box, Typography } from "@mui/material";
+import { FaCaretDown, FaCaretUp } from "react-icons/fa";
+
+type Prompt_view_arrowProps = {
+  PromptTemplate: string;
+};
+
+const Prompt_view_arrow: React.FC<Prompt_view_arrowProps> = ({
```

Comment:
@LovedeepSingh19 please following the naming convention as per other components. it should be camel cased

Reply:
Done👍🏻, Sorry about that.
<img width="1146" alt="Screenshot 2023-11-17 at 6 19 19 PM" src="https://github.com/sugarcane-ai/sugarcane-ai/assets/91473234/cfe7212f-d7d5-4fe2-8dc5-3d675de0a7be">
is this what you ment by "Just place that component above the variables text in the UI."